app/javascript/flavours/glitch/features/compose/components/options.jsx

```diff
@@ -83,8 +83,11 @@ const messages = defineMessages({
   },
 });
 
-@connect((state, { name }) => ({ checked: state.getIn(['compose', 'advanced_options', name]) }))
-class ToggleOption extends ImmutablePureComponent {
+const mapStateToProps = (state, { name }) => ({
+  checked: state.getIn(['compose', 'advanced_options', name]),
+});
+
+class ToggleOptionIntl extends ImmutablePureComponent {
```

Comment:
For some reason this has reverted back to `ToggleOptionIntl`

Reply:
No, this wasn't on purpose. Fixing it.